src/main/resources/config/liquibase/changelog/20240708144500_changelog.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="20240708144500" author="stoehrj">
+        <sql>
+            UPDATE competency_relation
+            SET type = type - 1
+            WHERE type > 0;
```

Comment:
The relation type is stored ordinal, so the first entry in the enum has value 0, the second entry has value 1 and so on.
Since we deleted the first entry in the enum (Relates), all other entries are automatically moved one forward and their value should be decreased by 1 as well.
The data base entries that already store 0 are now automatically converted to `Assumes`